File: 수박수박수박수박수박수?.py
"""
def solution(n):
    k = ""
    
    for i in range(n):
        if i % 2 == 0:
            k = k + "수"
        else:
            k = k + "박"
    return k
"""
#2022.11.12
def solution(n):
    return ('수박'*5000)[:n]

File: 수박수박수박수박수박수?.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(int n) {
    char* answer = (char*)malloc(n*3 + 1);
    if (n == 1){
        return "수";
    }
    strcpy(answer, "수박");
    for(int i = 2; i <= n/2; i++){
        strcat(answer, "수박");
    }
    if (n%2){
        strcat(answer, "수");
    }
    
    return answer;
}
